src/TIMER.CPP

```diff
@@ -36,7 +36,6 @@ static double tmrTicksToSecs(
 #if defined( TIMING_HIRES)
```

Comment:
Did you try building with TIMER_HIRES defined?

Reply:
@galanCA didn't we decide that we would add a comment next to the `#undef TIMING_HIRES` to indicate that it is no longer maintained?